src/game/game.h

```diff
@@ -552,17 +552,18 @@ struct clientstate
     int actortype, spawnpoint, ownernum, skill, points, frags, deaths, totalpoints, totalfrags, totaldeaths, spree, lasttimeplayed, timeplayed, cpmillis, cptime, queuepos;
     bool quarantine;
     string vanity;
-    vector<int> loadweap, lastweap, randweap;
+    std::vector<int> loadweap, lastweap, randweap;
```

Comment:
Again, is there a specific reason to change these? I'd just like to know your thinking.